random.c

```diff
@@ -421,7 +421,26 @@ random_init(int argc, VALUE *argv, VALUE obj)
 # define USE_DEV_URANDOM 0
 #endif
 
-#if USE_DEV_URANDOM
+#if HAVE_GETENTROPY
+# if defined(__ANDROID__) || defined(__APPLE__)
+#  include <sys/random.h>
+# endif
```

Comment:
There is the code for `<sys/random.h>` already, this should be there too.
That is, at line 52,
```C
#if defined HAVE_GETRANDOM || defined HAVE_GETENTROPY
```

Reply:
I just checked if I remove this line it fails to compile

Reply:
It means to add ` || defined HAVE_GETENTROPY` to line 52.

Reply:
Ah gotcha sorry.